### DISCUSSION TITLE
无法连接钱包

### DISCUSSION THREAD

**Discussion by taojun1979** [Chia-Network/chia-blockchain#5074] [Q&A]
无法连接钱包，永远是连接钱包的对话框。求助解决办法，谢谢！

**Comment by rupan-ji** [comment]
同样遇到这个问题，在一台MAC电脑上运行CHIA钱包1.1.5,永远在第一个白屏提示连接钱包，但我的另外一台笔记本上可以登录成功的
when open Chia wallet 1.1.5, always on sreen connet to the wallet, I use the MAC compute, but it success on my anther MACbook. Need help!
